docs/helpers/WebDriverIO.md

```diff
@@ -916,10 +856,8 @@ I.pressKey(['Control','a']);
 
 #### Parameters
 
--   `key` ([string][10] \| [Array][17]&lt;[string][10]>) key or array of keys to press.
-    
-
-    {{> \_keys }}To make combinations with modifier and mouse clicks (like Ctrl+Click) press a modifier, click, then release it.
+-   `key` **([string][10] | [Array][17]&lt;[string][10]>)** key or array of keys to press.
+    {{> _keys }}To make combinations with modifier and mouse clicks (like Ctrl+Click) press a modifier, click, then release it.
```

Comment:
should be `{{> _keys }}`  replaced by keys or link to keys?

Reply:
Ok!